scripts/clean-packages.sh

```diff
@@ -15,5 +15,5 @@ rm -r ${root_dir}/.dars 1> /dev/null 2>&1
 packages_yaml=${root_dir}/package/packages.yaml
 package_paths=($(yq e '.local.packages | to_entries | map(.value.package.path) | .[]' ${packages_yaml}))
 for package_path in "${package_paths[@]}"; do
-  daml clean --project-root ${root_dir}/package/${package_path}
+  dpm clean --project-root ${root_dir}/package/${package_path}
```

Comment:
Let's move back to `daml`. Migration to `dpm` should be the target of another PR


```suggestion
  daml clean --project-root ${root_dir}/package/${package_path}
```